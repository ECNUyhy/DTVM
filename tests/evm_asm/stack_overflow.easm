// Goal: Increase the stack depth to 1025,stack overflow
PUSH1 0x00
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
DUP1
