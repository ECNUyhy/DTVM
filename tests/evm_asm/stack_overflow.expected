{
  "status": "stack overflow",
  "error_code": 6,
  "stack": [],
  "memory": "",
  "storage": {},
  "transient_storage": {},
  "return": "",
  "events": []
}
